doc/classes/StreamPeerTCP.xml

```diff
@@ -29,6 +29,16 @@
 				Disconnect from host.
 			</description>
 		</method>
+		<method name="set_no_delay">
+			<return type="void">
+			</return>
+			<argument index="0" name="enabled" type="bool">
+			</argument>
+			<description>
+				Disables the Nagle algorithm to speed up the TCP protocol.
```

Comment:
I don't think this is a correct description of what Nagle algorithm does. If you don't need TCP_NODELAY then disabling it might make your overall application slower. I'd much prefer something like "Disable Nagle algorithm to improve latency for small packets. Note that for applications that send large packets, or need to transfer a lot of data, this can reduce total bandwidth."